x-pack/plugins/security_solution/public/common/components/ml_popover/hooks/use_enable_data_feed.ts

```diff
@@ -29,13 +36,17 @@ export const useEnableDataFeed = () => {
             indexPatternName: job.defaultIndexPattern,
             jobIdErrorFilter: [job.id],
             groups: job.groups,
+            prefix: installedJobPrefix(spaceId),
           });
+          enabledJobId = uninstalledJobIdToInstalledJobId(job.id, spaceId);
```

Comment:
Do we really need to compute the id to the installed version manually? It looks like the `setupMlJob` response contains the jobs installed with their actual id.

Reply:
Hmm, that is a good point. The tricky part is that it doesn't install a job but a module containing multiple jobs. So the response includes a list of jobs, and I would have to compute the id to find the installed job.  🫤